jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -1,5 +1,21 @@
 <%- # see https://bosh.io/docs/jobs/#properties for documentation of bosh ERB templates -%>
-<% if properties.ha_proxy.raw_config -%>
+<%
+  def format_indented_multiline_config(raw_config)
```

Comment:
This was bugging me for quite some time, thanks for taking the time to fix this 😄